apps/server/src/departments/departments.controller.spec.ts

```diff
@@ -1,20 +1,64 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { DepartmentsController } from './departments.controller';
 import { DepartmentsService } from './departments.service';
+import { PrismaService } from '../prisma/prisma.service';
+import { DepartmentEntity } from './entities/department.entity';
 
 describe('DepartmentsController', () => {
   let controller: DepartmentsController;
+  let departmentsService: DepartmentsService;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
       controllers: [DepartmentsController],
-      providers: [DepartmentsService],
+      providers: [DepartmentsService, PrismaService],
     }).compile();
 
     controller = module.get<DepartmentsController>(DepartmentsController);
+    departmentsService = module.get<DepartmentsService>(DepartmentsService);
   });
 
   it('should be defined', () => {
     expect(controller).toBeDefined();
   });
+
+  describe('findAll', () => {
```

Comment:
can we test the other routes as well?

Reply:
Made a new ticket. Will get to the testing later in the week hopefully.

Reply:
https://trello.com/c/1IraTXeL/46-departments-controller-tests